.github/ISSUE_TEMPLATE/config.yml.

```diff
@@ -0,0 +1,14 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Helios Design System support
+    url: https://helios.hashicorp.design/about/support
```

Comment:
This is not the same URL. I would differentiate between the generic support page (more friendly, easy to get context of where one is, and in case follow links to other support resources) and the specific support intake form, which is... not very friendly

Reply:
Ohhh, right. The go.hashi.co links to the Confluence intake form page 🤦🏻‍♀️